docs/resources/k8s_node_pool.md

```diff
@@ -38,7 +38,7 @@ resource "ionoscloud_ipblock" "example" {
 
 resource "ionoscloud_k8s_cluster" "example" {
   name                  = "k8sClusterExample"
-  k8s_version           = "1.20.10"
+  k8s_version           = "1.25.5"
```

Comment:
is this a requirement for private k8s?

Reply:
@cristiGuranIonos no, but it seems that `1.20.10` version is not available anymore:
```
Operation cannot be executed because the K8s version 1.20.10 is not viable for cluster.
```